bin/hummingbot_quickstart.py

```diff
@@ -100,25 +98,27 @@ async def quick_start(args):
         if not all_configs_complete(hb.strategy_name):
             hb.status()
 
-    with patch_stdout(log_field=hb.app.log_field):
-        dev_mode = check_dev_mode()
-        if dev_mode:
-            hb.app.log("Running from dev branches. Full remote logging will be enabled.")
-
-        log_level = global_config_map.get("log_level").value
-        init_logging("hummingbot_logs.yml",
-                     override_log_level=log_level,
-                     dev_mode=dev_mode)
-
-        if hb.strategy_file_name is not None and hb.strategy_name is not None:
-            await write_config_to_yml(hb.strategy_name, hb.strategy_file_name)
-            hb.start(log_level)
-
-        tasks: List[Coroutine] = [hb.run()]
-        if global_config_map.get("debug_console").value:
-            management_port: int = detect_available_port(8211)
-            tasks.append(start_management_console(locals(), host="localhost", port=management_port))
-        await safe_gather(*tasks)
+    class UIStartListener(EventListener):
+        def __call__(self, _):
+            asyncio.create_task(self.ui_start_handler())
+
+        @staticmethod
+        async def ui_start_handler():
+            dev_mode = check_dev_mode()
+            if dev_mode:
+                hb.app.log("Running from dev branches. Full remote logging will be enabled.")
+
+            if hb.strategy_file_name is not None and hb.strategy_name is not None:
+                await write_config_to_yml(hb.strategy_name, hb.strategy_file_name)
+                hb.start(global_config_map.get("log_level").value)
```

Comment:
@martinkou do we need to have the same class defined in two places? (in bin/hummingbot.py and in bin/hummingbot_quickstart.py)